tlatools/org.lamport.tlatools/src/util/SimpleFilenameToStream.java

```diff
@@ -309,24 +273,15 @@ public File resolve(String name) {
 	}
 
 	/**
-	 * Returns true iff moduleName is the name of a standard module.  It
-	 * assumes the empirically determined fact that libraryPaths is set to
-	 * have the path of the standard modules as the last element of the array.
+	 * Returns true iff moduleName is the name of a standard module.
```

Comment:
Add documentation regarding whether `.tla` suffix is expected.

Reply:
I made the minimal possible change to this comment to keep it valid. A clearer contract is in the top-level interface `FilenameToStream`:
```
     * Returns true iff <code>moduleName</code> resolves to a standard module.  The argument is treated the same as a
     * call to {@link #resolve(String, boolean)} with <code>isModule=true</code>.
```